code/modules/mob/living/carbon/alien/organs.dm

```diff
@@ -167,3 +167,144 @@
 	slot = ORGAN_SLOT_XENO_EGGSAC
 	w_class = WEIGHT_CLASS_BULKY
 	actions_types = list(/datum/action/cooldown/alien/make_structure/lay_egg)
+
+/// The stomach that lets aliens eat people/things
+/obj/item/organ/internal/stomach/alien
+	icon_state = "stomach-x"
+	w_class = WEIGHT_CLASS_BULKY
+	actions_types = list(/datum/action/cooldown/alien/regurgitate)
+	var/list/atom/movable/stomach_contents = list()
+
+/obj/item/organ/internal/stomach/alien/Destroy()
+	QDEL_LIST(stomach_contents)
```

Comment:
Is it possible to delete people by baking them into haggis?

Reply:
We don't appear to have haggis. We do have tsatsikh though.

Reply:
For that matter, exporting the stomach for a bounty might work.
As might any mutation toxin or other means of transformation.

Also. How does medical get them out if both they and the xeno are dead?

Reply:
Yes, it's also possible to delete people by eating the organ.
If they die in the stomach they are qdel'd.